shell/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -135,7 +135,7 @@ class ElectronDelegatedFrameHostClient
       const ElectronDelegatedFrameHostClient&) = delete;
 
   ui::Layer* DelegatedFrameHostGetLayer() const override {
-    return view_->GetRootLayer();
+    return view_->get_root_layer();
```

Comment:
same here

Reply:
Fixed in af772c5b047aa62658833db405857f13502919fa